tests/foreman/api/test_host.py

```diff
@@ -1438,3 +1438,45 @@ def test_positive_delete_and_check_host(self):
             host.read()
         except HTTPError:
             pytest.fail("HTTPError 404 raised unexpectedly!")
+
+
+class TestHostBulkAction:
+    """ Tests for host bulk actions. """
+
+    @pytest.mark.tier2
+    def test_positive_bulk_destroy(self, module_org):
+        """Destroy multiple hosts make sure that hosts were removed,
+        or were not removed when host is excluded from the list.
+
+        :id: 06d63376-8bf6-11eb-ab9f-98fa9b6ecd5a
+
+        :expectedresults: Included list of hosts,
+            that are not part of excluded list of host, are removed.
+
+        :CaseImportance: Medium
+        """
+
+        host_ids = []
+        for _ in range(3):
+            name = gen_choice(valid_hosts_list())
+            host = entities.Host(name=name, organization=module_org).create()
+            host_ids.append(host.id)
+
+        entities.Host().bulk_destroy(
+            data={
+                'organization_id': module_org.id,
+                'included': {'ids': host_ids},
+                'excluded': {'ids': host_ids[:-1]},
```

Comment:
included and excluded are required parameters. 
You need to specify which hosts you want to run it on. 
And you can pass empty list of excluded hosts if you want to run it only on included.

So if I understand correctly to this behaviour: mathematically,  union of Included - union of excluded =  result union
Both included and and excluded are required so this is not about priority :smile:

Reply:
Hm I was wondering why would someone want to spell out included hosts and then marked them as excluded... but I see there is also included[search] so that makes sense when you have a broad search but want to be sure not to remove a specific host, maybe a topic for extension PR